lib/file-utilities.js

```diff
@@ -248,8 +249,8 @@ const streamAllLogs = (config, callback) => {
             }
         });
 
-        journalctl.stderr.on('data', (err) => {
-            log('Error reading from journalctl: ' + err.toString().trim());
+        journalctl.stderr.on('data', (error) => {
+            log('error reading from journalctl: ' + error.toString().trim(), 'error');
```

Comment:
Could use a template literal here

Reply:
Actually, is there a reason for dropping everything to lower case for logging?

Reply:
Done!